webui/react/src/components/PaginatedNotesCard.module.scss

```diff
@@ -0,0 +1,95 @@
+.base {
+  display: flex;
+  height: 100%;
+  overflow: auto;
+
+  .sidebar {
+    flex-basis: 30%;
+    flex-shrink: 0;
+    max-width: 200px;
+    overflow-y: auto;
+    padding: 16px;
+
+    .listContainer {
+      display: flex;
+      flex-direction: column;
+      gap: 4px;
+      list-style: none;
+      margin: 0;
+      padding: 0;
+
+      .listItem {
+        align-items: center;
+        border: 2px solid transparent;
+        border-radius: 5px;
+        cursor: pointer;
+        display: flex;
+        padding: 6px;
+        width: 100%;
+
+        & > span {
+          flex-grow: 1;
+        }
+        .action {
+          visibility: hidden;
+        }
+        &:hover {
+          background-color: var(--theme-colors-monochrome-12);
+
+          .action {
+            visibility: visible;
+          }
+        }
+      }
+    }
+  }
+  .notesContainer {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    height: 100%;
+
+    & > * {
+      flex-grow: 1;
+    }
+  }
+}
+.emptyBase {
+  display: grid;
+  height: 100%;
+  padding: 40px;
+  place-content: center;
+  width: 100%;
+
+  .messageContainer {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: 18px;
+
+    & > * {
+      margin: 0;
+    }
+    & > i {
+      color: var(--theme-colors-monochrome-8)
+    }
+  }
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+  .base {
+    flex-direction: column-reverse;
+
+    .notesContainer {
+      min-height: 90%;
```

Comment:
**suggestion:** if the 10% is to make room for chrome or nav components, a percentage will be pretty brittle in terms of responsive design. A more fool proof method could be the use of `calc()` function to calculate exactly what's needed dynamically.